src/shogun/classifier/svm/GNPPLib.cpp

```diff
@@ -1,16 +1,8 @@
-/*-----------------------------------------------------------------------
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 3 of the License, or
- * (at your option) any later version.
- *
- * Library of solvers for Generalized Nearest Point Problem (GNPP).
- *
- * Written (W) 1999-2008 Vojtech Franc, xfrancv@cmp.felk.cvut.cz
- * Copyright (C) 1999-2008 Center for Machine Perception, CTU FEL Prague 
- *
--------------------------------------------------------------------- */
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
```

Comment:
this one should probably contain Vojtech Franc.
Tricky situation as he is not in the git log. Can you add him manually?